static/js/components/forms/ProfileFormFields.js

```diff
@@ -245,6 +259,7 @@ export const LegalAddressFields = ({
           component="select"
```

Comment:
Slightly unrelated but can you add a `htmlFor` attribute to the above `label`?

Reply:
added